src/Containers/AllProducts.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { useDispatch } from 'react-redux';
+import ProductComponent from './ProductComponents';
+import { myProducts } from '../Redux/Actions/productActions';
+
+const AllProducts = () => {
+  const dispatch = useDispatch();
+
+  const fetchProducts = async () => {
+    try {
+      const response = await axios.get('https://fakestoreapi.com/products');
+      dispatch(myProducts(response.data));
+    } catch (error) {
+      throw new Error('Error fetching products:', error);
+    }
+  };
+
+  useEffect(() => {
+    fetchProducts();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div>
+      <ProductComponent />
+    </div>
+  );
+};
+
```

Comment:
- [ ] Kindly pick one color for your design. You have many different colors on your homepage. Kindly choose one and stick to it.

  ![image](https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/36542986/15fb9849-6263-4538-8bfa-8e00acad474a)

- [ ] Kindly ensure you implement the alternating color pattern on the homepage as we have in the design. As you can see from the design there is a pattern between a lighter and darker theme of the chosen color. Kindly implement that.

  ![girl-2](https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/36542986/a560e5c4-0143-475a-98d3-d7ddd2ee4b65)

- [ ] Kindly ensure you implement the descriptive banner section as we have in the design.

  ![fdd](https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/36542986/c7126906-52a0-4c2b-bcfe-608f1651e2d1)

Reply:
Can be improved!